Source/Flow/Private/AddOns/FlowNodeAddOn.cpp

```diff
@@ -0,0 +1,78 @@
+// Copyright https://github.com/MothCocoon/FlowGraph/graphs/contributors
+
+#include "AddOns/FlowNodeAddOn.h"
+#include "Nodes/FlowNode.h"
+
+#include "Misc/RuntimeErrors.h"
+
+void UFlowNodeAddOn::InitializeInstance()
+{
+	CacheFlowNode();
+
+	Super::InitializeInstance();
+}
+
+void UFlowNodeAddOn::DeinitializeInstance()
+{
+	Super::DeinitializeInstance();
+
+	FlowNode = nullptr;
+}
+
+void UFlowNodeAddOn::TriggerFirstOutput(const bool bFinish)
+{
+	if (ensure(FlowNode))
+	{
+		FlowNode->TriggerFirstOutput(bFinish);
+	}
+}
+
+void UFlowNodeAddOn::TriggerOutput(const FName PinName, const bool bFinish , const EFlowPinActivationType ActivationType)
+{
+	if (ensure(FlowNode))
+	{
+		FlowNode->TriggerOutput(PinName, bFinish, ActivationType);
+	}
+}
+
+void UFlowNodeAddOn::Finish()
+{
+	if (ensure(FlowNode))
+	{
+		FlowNode->Finish();
+	}
+}
+
+EFlowAddOnAcceptResult UFlowNodeAddOn::AcceptFlowNodeAddOnParent_Implementation(const UFlowNodeBase* ParentTemplate) const
```

Comment:
This is the function on the AddOn-side to make itself eligible to a parent Node or AddOn that knows nothing about it. 

This case is useful for AddOns in plugins to well-established FlowNodes, where we don't want to modify the Node to add eligibility checks for the new AddOn.